java/build.sh

```diff
@@ -9,7 +9,7 @@ mkdir -p target/classes/META-INF/lib
 
 swig -c++ -java -package org.rismch.verovio -outdir src/main/java/org/rismch/verovio verovio.i
 
-SRCFILES=`LS ../src/*.cpp`
+SRCFILES=`ls ../src/*.cpp`
```

Comment:
FYI, I'm packaging verovio for the Mageia distribution, and I may submit other comments due to what I find doing this.